.travis.yml

```diff
@@ -0,0 +1,19 @@
+notifications:
+  email: false
+
+sudo: required
+
+services:
+  - docker
+
+git:
+  depth: false
+
+script:
+  - docker build . -t renku/renku-demo
+
+after_success:
```

Comment:
in the other repos, we added a deploy step -- see https://github.com/SwissDataScienceCenter/renku/blob/master/.travis.yml#L80

I think this will work more or less the same but perhaps the deploy is cleaner?

Reply:
I avoided the deploy step because I found it kind of confusing to have a deploy step when there's nothing deployed (or maybe I have a wrong understanding of the word 'deploy'?). Anyway, I can make it a deploy step if you really think it's better.

Reply:
right, I guess "deploy" is used rather loosely. An additional problem with the configuration here is that it will try to push the image on builds in forks. Since we're using deploy everywhere else, it should also be used here.

Reply:
Ok, I'll adapt it. Just out of curiosity: does this mean that travis omits the deploy step on forks?

Reply:
it should afaik